service/qos/service_level_controller.cc

```diff
@@ -212,6 +217,27 @@ future<> service_level_controller::update_service_levels_from_distributed_data()
     });
 }
 
+void service_level_controller::reload_configuration() {
+    if (!_global_controller_db) {
+        return;
+    }
+    if (_global_controller_db->configuration_reload_aborter.abort_requested()) {
+        return;
+    }
+
+    _global_controller_db->configuration_reload = _global_controller_db->configuration_reload.then([this] () -> future<> {
```

Comment:
I'm wrong here.
`group0_state_machine` should do `co_await update_service_levels_from_distributed_data()` to make sure doing read barrier is equal to updating service levels configuration.